include/Makefile

```diff
@@ -2,22 +2,23 @@ SHELL=/bin/bash
 
 DIRNAME=$(shell basename "$(PWD)")
 META=meta.ily
+DEPLOY=../../../tools/deploy.sh
```

Comment:
Maybe better `../../../tools/deploy.sh --p = $(DIRNAME)`?  I think passing the dirname directly to the python script is the easiest way to determine the github-link for the deplyoment script. I don't know how hard it would be to determine (re-engineer) the value of $DIRNAME from inside `deploy.sh`, if we don't pass it as parameter.

Reply:
introduced a new variable DEPLOY_PARAMS which is used when calling the deployment-tool.